src/main/java/org/folio/rest/impl/PurchaseOrdersAPI.java

```diff
@@ -38,7 +38,9 @@
 import io.vertx.ext.web.handler.HttpException;
 
 public class PurchaseOrdersAPI extends BaseApi implements OrdersStoragePurchaseOrders {
+
   private static final Logger log = LogManager.getLogger();
+  private static final String ORDER_ID_WHERE_CLAUSE = "WHERE jsonb ->> 'purchaseOrderId' = '%s'";
```

Comment:
There is foreign key exists for purchaseOrderId in this table, using it will improve performance. If it very complex change, then can change in subsequnet PR